source/elements/oneDAL/source/glossary.rst

```diff
@@ -0,0 +1,170 @@
+
+    Inference set
+        An unlabeled :term:`dataset` used at the :term:`inference` stage to make an outputs from the :term:`model`.
+
+    Label
+        A :term:`feature` of some object or event which value is target to be predicted in :term:`supervised learning` task.
+        On the :term:`training` stage, the model which finds the best dependency between input :term:`feature vector` and
+        :term:`labels<Label>` is selected. :term:`Label` is one of outputs can be made by :term:`model`
+        on the :term:`inference` stage.
+
+    Model
+        A result of :term:`training` machine-learning algorithm that stores information necessary to run
+        :term:`inference` on a new :term:`dataset`.
+
+    Observation
+        A vector of at least one :term:`feature` value and zero or more :term:`label<Label>` values.
+        Each of :term:`labels<Label>` and :term:`features<Feature>` can encode different objects or events.
+
+        Synonyms: instance, sample.
+
+    Ordinal feature
+        A :term:`feature` with a discrete set of possible values and some ordering between them.
+
+    Regression
+        A :term:`supervised machine-learning problem<Supervised learning>` of assigning
+        :term:`continuous<Continuous feature>` :term:`labels<Label>` to
+        :term:`feature vectors<Feature vector>`.
+
+    Response
+        A value of a single :term:`label`. For example, response may be a flag
+        indicating that an observation belongs to a particular class.
+
+    Supervised learning
+        :term:`Training` process with :term:`labeled<Label>` :term:`training set`.
+        The best :term:`model` of dependencies between input :term:`feature vector`
+        and :term:`labels<Label>` is selected.
+
+    Training
+        A process of creating a best :term:`model` (in terms of some quality criteria) based on
+        information extracted from a :term:`training set`.
+
+    Training set
+        A :term:`dataset` used at the :term:`training` stage to create a :term:`model`.
+
+    Unsupervised learning
+        :term:`Training` process where :term:`training set` does not contain information
+        of dependencies between :term:`feature vectors<Feature vector>` and outputs need
+        to be produced by the :term:`model`.
+
+|dal_short_name| terms
+======================
+
+.. glossary::
+    :sorted:
+
+    Batch Mode
+        The computation mode for an algorithm in |dal_short_name|, where all the data needed for
+        computation is available at the start and fits the memory of the device on which the computations are performed.
+
+    Contiguous data
+        A special property of the :term:`data format`: the data are stored as one contiguous memory block.
+
+    Data format
+        The representation of internal structure of the data: this information is enough to efficiently access them.
+
+    Data type
+        An integral or floating-point type which is used to physically store the data.
```

Comment:
what term is used by c++ standard, integral, basic, scalar?

Reply:
- *character types` for `char`, `unsigned char`, ...
- *integer types* for `int`, `long`, `short` ...
- *integral types* for `bool` + character types + integer types